dev/test_balancing_calc

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#--------------------------------------------------------------------
+
+# Helps to calculate new balancing
```

Comment:
Nit: Since this is outside of the unit test layer, it would be nice to elaborate a bit on what we mean by balancing here.

Reply:
this has no use for everyday developers; its only a tool for optimizing the workflow....so I've moved it to be under `.github/scripts` ; eventually this should be replaced I guess with something better...

it feels awkward to work with these script; maybe java or groovy would be better